<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectConnectionDialogBase</class>
 <widget class="QDialog" name="SelectConnectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select saved PostgreSQL connection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/svedigark.svg</normaloff>assets/svedigark.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="PSQLConnectionLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Postgresql connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="postGisConnectionComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>select working postgres connection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="editServerHost">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Port">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Username">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_Server">
         <property name="text">
          <string>Host name/-adress</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editServerPort">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editUsername">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_Password">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QgsPasswordLineEdit" name="editPassword">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="help_button">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
